addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1473,7 +1473,7 @@ export class OdooEditor extends EventTarget {
         const OFFSET = 10;
         let isBottom = false;
         this.toolbar.classList.toggle('toolbar-bottom', false);
-        this.toolbar.style.maxWidth = this.editable.offsetWidth - OFFSET * 2 + 'px';
+        this.toolbar.style.maxWidth = window.innerWidth - OFFSET * 2 + 'px';
```

Comment:
Did you test in mass mailing? Just want to make sure we avoid wrong window reference issues.

Reply:
Mass mailing doesn't have the floating toolbar, and it's the only place we use the Iframe version of the wysiwyg editor.

So it should never be a problem.